projects/plugins/jetpack/_inc/lib/debugger/class-jetpack-debug-data.php

```diff
@@ -133,6 +133,8 @@ public static function core_debug_data( $debug ) {
 	 * }
 	 */
 	public static function debug_data() {
+		$blog_key   = null;
+		$user_key   = null;
```

Comment:
Kind of pointless when they're only ever used in a code path where they'll always be set. If we do want to pre-set them, it'd be better to pre-set them to `'Not set.'` and then remove the conditionals on lines 219 and 224.

Plus it would make more sense to put these down near line 188 instead of all the way up here.

Reply:
marking as a false positive.